README.md

```diff
@@ -1,7 +1,6 @@
 # Source Engine Query Cacher
 
-## This is a hard fork of
-https://github.com/hyperxpro/SourceEngineQueryCacher
+## This is a hard fork of https://github.com/hyperxpro/SourceEngineQueryCacher
```

Comment:
This probably has to be reworked if you decide to approve the PR. I wasn't certain if you were interested so I made sure people get referred to you if they stumble across this on other platforms like GitLab